fedireads/models/import_job.py

```diff
@@ -67,19 +67,19 @@ def get_book_from_db_isbn(self):
 
     def get_book_from_isbn(self):
```

Comment:
Can ```get_book_from_db_isbn``` go away now? This should now search both sources in the correct order?

Reply:
yes